src/briefcase/commands/dev.py

```diff
@@ -207,8 +206,8 @@ def venv_name(self) -> str:
 
         :returns: Name for virtual environment directory
         """
-
-        return "dev"
+        python_version = f"{sys.version_info.major}.{sys.version_info.minor}"
+        return f"dev--{python_version}"
```

Comment:
Assuming this is a typo
```suggestion
        return f"dev-{python_version}"
```